sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -972,10 +972,9 @@ case class ShowColumnsCommand(
  */
 case class ShowPartitionsCommand(
     tableName: TableIdentifier,
+    attributes: Seq[Attribute],
```

Comment:
why not just name this parameter `output`?

Reply:
> why not just name this parameter `output`?

Never wrote like this before, lookup the code, some places write like that. Update